client/src/context/useBookingContext.tsx

```diff
@@ -0,0 +1,25 @@
+import { useContext, createContext, FunctionComponent, useState, useEffect } from 'react';
+import { Booking } from '../interface/Booking';
+import sortedBookings from '../helpers/utils/sortBookings';
+
+interface IBookingContext {
+  bookings: Booking[];
+}
+
+export const BookingContext = createContext<IBookingContext>({
+  bookings: sortedBookings,
+});
+
+export const BookingProvider: FunctionComponent = ({ children }): JSX.Element => {
+  const [bookings, setBookings] = useState<Booking[]>([]);
+
+  useEffect(() => {
+    setBookings(sortedBookings);
+  }, [bookings]);
+
+  return <BookingContext.Provider value={{ bookings }}>{children}</BookingContext.Provider>;
+};
+
+export function useRequest(): IBookingContext {
+  return useContext(BookingContext);
+}
```

Comment:
Nice job with the context 👍🏼 Please double check the `useEffect` depedency array for me🙏🏼